test_data/yast/raid1_lvm/lvm_raid1_opensuse.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
file name: `test_data/yast/lvm+raid1/lvm+raid1_opensuse.yaml`

Reply:
test_data/yast/raid1_lvm/lvm_raid1_opensuse.yaml > test_data/yast/raid1_lvm/lvm+raid1_opensuse.yaml

let s not change the directory's name again